workflows/rnavar.nf

```diff
@@ -0,0 +1,401 @@
+include { PREPARE_GENOME                } from '../subworkflows/local/prepare_genome'           // Build the genome index and other reference files
+include { SPLITNCIGAR                   } from '../subworkflows/local/splitncigar'              // Splits reads that contain Ns in their cigar string
+include { ANNOTATE                      } from '../subworkflows/local/annotate'                 // Annotate variants using snpEff or VEP or both
+
+/*
+========================================================================================
+    IMPORT NF-CORE MODULES
+========================================================================================
+*/
+
+include { FASTQC                        } from '../modules/nf-core/modules/fastqc/main'
+include { MULTIQC                       } from '../modules/nf-core/modules/multiqc/main'
+include { CAT_FASTQ                     } from '../modules/nf-core/modules/cat/fastq/main'
+include { GATK4_BASERECALIBRATOR        } from '../modules/nf-core/modules/gatk4/baserecalibrator/main'
+include { GATK4_BEDTOINTERVALLIST       } from '../modules/nf-core/modules/gatk4/bedtointervallist/main'
+include { GATK4_INTERVALLISTTOOLS       } from '../modules/nf-core/modules/gatk4/intervallisttools/main'
+include { GATK4_HAPLOTYPECALLER         } from '../modules/nf-core/modules/gatk4/haplotypecaller/main'
+include { GATK4_MERGEVCFS               } from '../modules/nf-core/modules/gatk4/mergevcfs/main'
+include { GATK4_INDEXFEATUREFILE        } from '../modules/nf-core/modules/gatk4/indexfeaturefile/main'
+include { GATK4_VARIANTFILTRATION       } from '../modules/nf-core/modules/gatk4/variantfiltration/main'
+include { SAMTOOLS_INDEX                } from '../modules/nf-core/modules/samtools/index/main'
+include { CUSTOM_DUMPSOFTWAREVERSIONS   } from '../modules/nf-core/modules/custom/dumpsoftwareversions/main'
+
+/*
+========================================================================================
+    IMPORT NF-CORE SUBWORKFLOWS
+========================================================================================
+*/
+
+include { ALIGN_STAR                    } from '../subworkflows/nf-core/align_star'         // Align reads to genome and sort and index the alignment file
+include { MARKDUPLICATES                } from '../subworkflows/nf-core/markduplicates'     // Mark duplicates in the BAM file
+include { RECALIBRATE                   } from '../subworkflows/nf-core/recalibrate'        // Estimate and correct systematic bias
+
+/*
+========================================================================================
+    VARIABLES
+========================================================================================
+*/
+
+// Check STAR alignment parameters
+def prepareToolIndices  = params.aligner
+def seq_platform        = params.seq_platform ? params.seq_platform : []
+def seq_center          = params.seq_center ? params.seq_center : []
```

Comment:
What is the difference between `seq_platform` and `seq_center`?

Reply:
PL and CN respectively in RG fields. These go to `STAR/align` and PL is needed in the BAM header for GATK downstream analysis.

Reply:
we have `params.sequencing_center` in sarek.
But I can see that they're using `params.seq_center` in rnaseq.
@FriederikeHanssen should we switch to `params.seq_center` to have more cohesion?

Reply:
sure, I am all for similar names all over the place. Do we also need platform?

Reply:
I think platform is eg. "illumina HiSeq" and center is "QBiC" so they are different things. Not sure if / how much they're used.

Reply:
yeah turns out ILLUMINA was hard coded in sarek so never thought of it. For sarek we decidded now to default to ILLUMINA and otherwise let the user set something. Seems sensitive to me, not sure anyone is using this, but maybe there is a use case for it